src/monopole_gr/monopole_gr.cpp

```diff
@@ -57,6 +61,17 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
   int nth_call = 0;
   params.Add("nth_call", nth_call, true); // mutable
 
+  // Time step control. Equivalent to CFL
+  // Empirical. controls
+  Real dtfac = pin->GetOrAddReal("monopole_gr", "dtfac", 0.9);
+  params.Add("dtfac", dtfac);
+
+  // Warn if the change in lapse is too different from dalpha/dt
+  bool dtwarn = pin->GetOrAddBoolean("monopole_gr", "warn_on_dt", false);
```

Comment:
A little confusing that `dtwarn` is the name in the code and `warn_on_dt` is the name in the input file.